src/api/app/views/webui2/webui/monitor/_workers_table.html.haml

```diff
@@ -1 +1,49 @@
-%h2 Workers
+#workers-form
+  %form{ action: '/' }
+    %p
+      %div
+        %span.strong Show:
+        = select_tag(:worker_display, options_for_select([['Package name', 'package'], ['Project', 'project'],
```

Comment:
I miss here the class `custom-select` to be consistent with the other selects, like in the graph section.
[patterns](https://obs-patterns.netlify.com/molecules/forms.html)

Reply:
done, screenshot attached